### ISSUE TITLE
[Global Styles] Orange warning button's colors are not accessible

### ISSUE THREAD

**Issue by andycochran** [zurb/foundation-sites#7378]
**How can we reproduce this bug?**
Go to Global Styles Page
Use WebAIM Color Contrast Checker (http://webaim.org/resources/contrastchecker)
Enter "FFFFFF" as "foreground" color.
Enter "ffae00" as "background" color.

**What did you expect to happen?**
Contrast ratio for normal text should be 4.5:1 or higher
WCAG AA criteria should pass

**What happened instead?**
Contrast ratio is below standard.
See: http://webaim.org/resources/contrastchecker/?fcolor=ffffff&bcolor=ffae00

**Comment by andycochran** [comment]
Aesthetically, this one is tough. In order to make this hue accessible, it has to be darkened to an ugly brown. An alternative solution would be to darken the text to `#0a0a0a` (same as body color) or `#463000` (same hue as button background, but darkened). Unfortunately, then it's the only button with dark text (unless the same solution is used for the currently-inaccessible "disabled" buttons).

**Comment by gakimball** [comment]
I'm collecting all of the contrast issues into one: #7387
